src/phylum/ci/lockfile.py

```diff
@@ -1,10 +1,16 @@
 """Define a lockfile implementation.
 
 A Phylum project can consist of multiple lockfiles.
-This class represents a single lockfile.
+This module/class represents a single lockfile.
+
+For historical purposes, this module uses "lockfile" language instead of the
+more general term "dependency file." Both lockfiles and manifests are supported
```

Comment:
```suggestion
more general term "dependency file". Both lockfiles and manifests are supported
```

Reply:
Ooh...interesting. I think the difference in punctuation placement here is due to my use of American English. [It appears](https://www.thesaurus.com/e/grammar/does-punctuation-go-inside-or-outside-quotation-marks/#when-to-use-outside-of-punctuation-marks) that British English puts the punctuation outside of the quotes. Personally, I don't like the way sentences look when they end with a quote and so I usually try to re-write them to avoid the situation entirely. I'll do that here:

```suggestion
more general "dependency file" term. Both lockfiles and manifests are supported
```

Reply:
Yeah I was aware of this difference and contemplating to just not add my comment. But just *logically* speaking it makes no sense to me to have the punctuation inside the quote, and I cannot stand looking at it. :smile: